docs/Configuration.md

```diff
@@ -188,127 +260,169 @@ return {
 ```
 
 ### `moduleFileExtensions` [array<string>]
+
 Default: `["js", "json", "jsx", "node"]`
 
-An array of file extensions your modules use. If you require modules without specifying a file extension, these are the extensions Jest will look for.
+An array of file extensions your modules use. If you require modules without
+specifying a file extension, these are the extensions Jest will look for.
 
-If you are using TypeScript this should be `["js", "jsx", "json", "ts", "tsx"]`, check [ts-jest's documentation](https://github.com/kulshekhar/ts-jest).
+If you are using TypeScript this should be `["js", "jsx", "json", "ts", "tsx"]`,
+check [ts-jest's documentation](https://github.com/kulshekhar/ts-jest).
 
 ### `moduleDirectories` [array<string>]
+
 Default: `["node_modules"]`
 
-An array of directory names to be searched recursively up from the requiring module's location. Setting this option
-will _override_ the default, if you wish to still search `node_modules` for packages include it
-along with any other options: `["node_modules", "bower_components"]`
+An array of directory names to be searched recursively up from the requiring
+module's location. Setting this option will _override_ the default, if you wish
+to still search `node_modules` for packages include it along with any other
+options: `["node_modules", "bower_components"]`
+
+### `moduleNameMapper` [object<string,
+string>]
 
-### `moduleNameMapper` [object<string, string>]
 Default: `null`
 
-A map from regular expressions to module names that allow to stub out resources, like images or styles with a single module.
+A map from regular expressions to module names that allow to stub out resources,
+like images or styles with a single module.
 
-Modules that are mapped to an alias are unmocked by default, regardless of whether automocking is enabled or not.
+Modules that are mapped to an alias are unmocked by default, regardless of
+whether automocking is enabled or not.
 
-Use `<rootDir>` string token to refer to [`rootDir`](#rootdir-string) value if you want to use file paths.
+Use `<rootDir>` string token to refer to [`rootDir`](#rootdir-string) value if
+you want to use file paths.
 
-Additionally, you can substitute captured regex groups using numbered backreferences.
+Additionally, you can substitute captured regex groups using numbered
+backreferences.
 
 Example:
+
 ```json
 {
   "moduleNameMapper": {
     "^image![a-zA-Z0-9$_-]+$": "GlobalImageStub",
-    "^[./a-zA-Z0-9$_-]+\.png$": "<rootDir>/RelativeImageStub.js",
-    "module_name_(.*)": "<rootDir>/substituted_module_$1.js"
-  }
+    "^[./a-zA-Z0-9$_-]+.png$": "<rootDir>/RelativeImageStub.js",
```

Comment:
The removal of the `\` is safe, I guess?

Reply:
Should it be `\\`?

Reply:
Yeah it probably found a bug in your regex :P